storage/innobase/log/log0log.cc

```diff
@@ -928,8 +940,10 @@ void log_print(const log_t &log, FILE *file) {
     checkpoint age */
     fprintf(file,
             "Log tracking enabled\n"
-            "Log tracked up to   " LSN_PF "\n",
-            log_sys->tracked_lsn.load());
+            "Log tracked up to            " LSN_PF
+            "\n"
+            "Max tracked LSN age          " LSN_PF "\n",
+            log_sys->tracked_lsn.load(), max_checkpoint_age);
```

Comment:
Do we need this duplicated print? Did it exist in 5.7?

Reply:
@satya-bodapati yes, added here for consistency - https://github.com/percona/percona-server/blob/Percona-Server-5.7.31-34/storage/innobase/log/log0log.cc#L2918

Reply:
ok, lets keep it then